src/services/request-service.ts

```diff
@@ -0,0 +1,75 @@
+import Promise = require('bluebird');
+import r = require('rethinkdb');
+import uuid = require('uuid');
+import EmailService = require('./email-service');
+import nodemailer = require('nodemailer');
+import q = require('../dbutils/query');
+import models = require('../models/models');
+import errors = require('../errors/errors');
+import config = require('../config');
+
+
+module RequestService {
```

Comment:
Yeah, I saw that last night after I posted this, along with some other things we can try.  I will try them out when I get out of class at 2.